src/stokes/Stokes2D.jl

```diff
@@ -545,14 +546,13 @@ function _solve!(
         Aij .= 0.0
     end
 
-
     # compute buoyancy forces and viscosity
     compute_ρg!(ρg, phase_ratios, rheology, args)
     compute_viscosity!(stokes, phase_ratios, args, rheology, viscosity_cutoff)
     displacement2velocity!(stokes, dt, flow_bcs)
 
     while iter ≤ iterMax
-        iterMin < iter && ((err / err_it1) < ϵ_rel || err < ϵ_abs) && break
+        # iterMin < iter && ((err / err_it1) < ϵ_rel && err < ϵ_abs) && break
```

Comment:
```suggestion
        # iterMin < iter && ((err / err_it1) < ϵ_rel || err < ϵ_abs) && break
```
this needs to be like this. otherwise it doesn't break if the relative tolerance is below its set value but the abs is not. same vise versa. 
And I think it shouldn't be commented as there's no break condition anymore.